extensions/MyDashboard/template/en/default/pages/mydashboard.html.tmpl

```diff
@@ -108,7 +108,7 @@
         <span id="query_count_refresh" class="bz_default_hidden">
           <span class="items_found" id="query_bugs_found">0 [% terms.bugs %] found</span>
           | <a class="refresh"     href="javascript:void(0);" id="query_refresh">Refresh</a>
-          <span id="bar_markread">|</span> <a class="markread" href="javascript:void(0);" id="query_markread">none</a>
+          <span id="bar_markread">|</span> <a class="refresh" href="javascript:void(0);" id="query_markread">none</a>
```

Comment:
This change is absolutely not needed. Probably some mistake on my part while committing.